Obedient-Lobsters/WorkforceManagement
# Department: Add a new Department

# Description
The user can now navigate to Create Department form by clicking the "Create New" link. The form allows the user to add a department name and is posted to the database when the user clicks "Create" button. The user is redirected to view all department page once the form is completed. Update to README was accidentally completed in a previous pull request.
 Fixes #4 
 ## Type of change
 # Testing Instructions
If you need to create the database:

Start SQL Server Management Studio

Run BangazonDatabaseCreateSeed.sql in SSMS

Update the appsetting.json in Visual Studio community to reflect your desktop and your server

If your database is already seeded or has completed the steps above:

In the terminal, `git fetch --all` and `git checkout ss-AddDepartment` to switch branches

Start the program by cd'ing into the WorkforceManagement (make sure you are deep enough, the .sln file is a few WorkforceManagement directories deep) and using the command `start WorkforceManagement.sln`.

Press F5 to start the program in Visual Studio

Once the program is running, navigate to your browser and go to the localhost address shown in the command prompt that opens up when you run the program. Click on the Departments link in the navbar, and you should see a list with all Departments in the database.

Click on the "Create New" link. Fill the form out with a new department name and click "Create". A new department is created in the database and you should be rerouted back to view all list will the new department listed.

 # Checklist:
